paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -365,6 +389,9 @@ void CPUQuantizePass::QuantizeConcat(Graph* graph) const {
     // get scales calculated after warmup, they scale variables to MAX=1.0
     auto scales = Get<VarQuantScale>("quant_var_scales");
 
+    // make sure all the required scales are available
+    CheckScalesForVariables({concat_out->Name()});
```

Comment:
Shouldn't concat inputs also be checked?

Reply:
No, when quantizing `concat`, we use only the scale for output. The output scale is being calculated based on the inputs during scale gathering.

Reply:
That's right, I've completely forgotten. Thanks!